tests/sklearn/test_sklearn_models.py

```diff
@@ -1763,3 +1836,43 @@ def test_linear_models_have_no_tlu(
 
     # Check that no TLUs are found within the MLIR
     check_circuit_has_no_tlu(fhe_circuit)
+
+
+# Test only tree-based models
+@pytest.mark.parametrize("model_class, parameters", get_sklearn_tree_models_and_datasets())
+@pytest.mark.parametrize("n_bits", [3, 5, 6])
+def test_rounding_consistency(
```

Comment:
This test seems to be ~13 minutes long. I propose to only use the (new) simulation instead of FHE. WDYT?

Reply:
maybe we only need to test n_bits=5 ?

Reply:
Still 5 minutes for one test is a bit long. Do we really need to run this in FHE?

Reply:
Rounding is applied when the circuit has a bit width greater than 4. I set n_bits to 3 to ensure complete coverage

I agree 5 mn is long for a test, but it's the only way to test that we have the same accuracy.

Reply:
for me we should : 
- do in simulation in regular PRs
- do in FHE in weekly 

that's what we usually do !